nebula_hw_interfaces/src/nebula_velodyne_hw_interfaces/velodyne_hw_interface.cpp

```diff
@@ -167,13 +167,44 @@ VelodyneStatus VelodyneHwInterface::check_and_set_config(
               << std::endl;
   }
 
-  target_key = "config.fov.start";
-  auto current_cloud_min_angle = tree.get<std::uint16_t>(target_key);
   int setting_cloud_min_angle = sensor_configuration->cloud_min_angle;
-  // Velodyne only allows a maximum of 359 in the setting
-  if (setting_cloud_min_angle == 360) {
-    setting_cloud_min_angle = 359;
+  int setting_cloud_max_angle = sensor_configuration->cloud_max_angle;
+
+  // FIXME: VLP16 has problems for timestamp. Whatch github issue #
```

Comment:
Please explain in more detail (combination of hardware and software scan cutting causes timestamping issues at the FoV borders, so the hardware FoV has to be increased for scan cutting to work.